src/pages/index.js

```diff
@@ -1,66 +1,98 @@
 const editCloseButton = editProfileModal.querySelector(".modal__close-button");
 const profileFormElement = editProfileModal.querySelector(".modal__form");
 const nameInput = editProfileModal.querySelector("#name-input");
 const jobInput = editProfileModal.querySelector("#description-input");
-const profileNameElement = document.querySelector(".profile__name");
-const profileJobElement = document.querySelector(".profile__description");
-const cardTemplate = document.querySelector("#card-template");
-const cardsList = document.querySelector(".cards__list");
-const newPostModal = document.querySelector("#new-post");
+
+//Edit avatar modal elements
+const editAvatarButton = document.querySelector(".profile__avatar-button");
+const editAvatarModal = document.querySelector("#edit-avatar");
+const avatarSubmitButton = editAvatarModal.querySelector(
+  ".modal__submit_button"
+);
+const avatarCloseButton = editAvatarModal.querySelector(".modal__close-button");
+const avatarInput = editAvatarModal.querySelector("#avatar-input");
+const avatarForm = editAvatarModal.querySelector(".modal__form");
+
+//New post modal elements
 const newPostButton = document.querySelector(".profile__post-button");
+const newPostModal = document.querySelector("#new-post");
 const newPostCloseButton = newPostModal.querySelector(".modal__close-button");
 const newPostForm = newPostModal.querySelector(".modal__form");
 const linkInput = newPostModal.querySelector("#link-input");
 const captionInput = newPostModal.querySelector("#caption-input");
+const newPostSubmitButton = newPostModal.querySelector("#new-post-submit");
+
+//Post preview modal elements
 const previewModal = document.querySelector("#preview-modal");
 const previewImage = previewModal.querySelector(".modal__image");
 const previewCaption = previewModal.querySelector(".modal__image-caption");
 const previewCloseButton = previewModal.querySelector(
   ".modal__close-button_type_preview"
 );
-const newPostSubmitButton = document.querySelector("#new-post-submit");
 
+//Profile elements
+const avatarImage = document.querySelector(".profile__avatar");
+const profileNameElement = document.querySelector(".profile__name");
+const profileJobElement = document.querySelector(".profile__description");
+
+//Card elements
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+let selectedCard;
+let selectedCardId;
+
+//API initialization
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "36b3b370-c3f7-447d-abe8-f5174d0c50a3",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, me]) => {
+    cards.forEach((card) => {
+      const cardFill = getCardTemplate(card);
+      cardsList.prepend(cardFill);
+    });
+    const userInfo = me;
+    const profileAvatar = document.querySelector(".profile__avatar");
```

Comment:
`document.querySelector(".profile__avatar")`

This element has already been found in line 59 as `avatarImage`. Please, use it here